packages/data-explorer/src/store/actions.ts

```diff
@@ -56,7 +55,23 @@ export default {
       commit('applyBookmark')
     }
   },
-  fetchCardViewData: async ({ commit, state, getters }: { commit: any, state: ApplicationState, getters: any }) => {
+  async fetchViewData (store, { tableName, pagination }) {
+    if (store.state.tableName !== tableName) {
+      await store.dispatch('fetchTableMeta', { tableName })
+      if (store.getters.isUserAuthenticated) {
+        await store.dispatch('header/fetchBreadcrumbs', null, { root: true })
```

Comment:
in general passing 'null' seems like a code smell, in this case not passing the getters seems to lead to breaking the breadcrumbs

Reply:
This also works:
```
await store.dispatch('header/fetchBreadcrumbs')
```